src/components/07-form/controls/radio-buttons.njk

```diff
@@ -19,3 +19,25 @@
     </div>
   </fieldset>
 </form>
+
+<form class="usa-form">
+  <fieldset class="usa-fieldset">
+    <legend class="usa-legend usa-legend">Same list as radio tiles</legend>
```

Comment:
Thoughts on including this as the same list with this update to the `legend`? The guidance will essentially be the same between the two variants, so it might make sense to keep them together. We can simply call out what instances this variant is optimized for, and that lists should not mix the variants.

Reply:
@thisisdano Here's how it looks on the documentation site. The `legend`s have a tall top margin, so I used a utility on the `h6` to create some space.
<img width="650" alt="Screen Shot 2020-10-28 at 2 37 39 PM" src="https://user-images.githubusercontent.com/1094951/97481438-35183880-192b-11eb-9c29-fcd5660f19a6.png">

Reply:
Yeah, that's conceptually right, even if the spacing is too loose

Reply:
@thisisdano perhaps add a negative margin to the `h6` on the documentation site to pull the fieldsets up?